src/main/java/io/airbrake/javabrake/Constant.java

```diff
@@ -1,10 +1,23 @@
 package io.airbrake.javabrake;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class Constant {
 
-    public static String apmRoute = "routes-stats";
-    public static String apmRouteBreakDown = "routes-breakdowns";
-    public static String apmQuery = "queries-stats";
-    public static String apmQueue = "queues-stats";
+    protected static String apmRoute = "routes-stats";
+    protected static String apmRouteBreakDown = "routes-breakdowns";
+    protected static String apmQuery = "queries-stats";
+    protected static String apmQueue = "queues-stats";
+    protected static int maxRetryAttempt = 1;
+    private static Integer[] statusCode = { 404, 408, 409, 410, 500, 502, 504};
+    protected static int BACKLOG_FLUSH_PERIOD = 60;
+    protected static List<Integer> getStatusCodeCriteriaForBacklog() {
```

Comment:
This doesn't express that these are failure codes. What do you think of renaming this one to something like `failureCodeList()` instead.

Reply:
ok